controller/userController.go

```
@@ -57,20 +59,20 @@ func (u *UserController) RegisterEndpoints(api *gin.RouterGroup) {
 func (u *UserController) get(c *gin.Context) {
 	userUuid, err := uuid.Parse(c.Param("uuid"))
 	if err != nil {
-		zap.S().Errorf("error parsing uuid value = %s", c.Param("uuid"))
+		u.logger.Errorf("error parsing uuid value = %s", c.Param("uuid"))
 		c.AbortWithError(http.StatusBadRequest, err)
```

golangci-lint (1.64.8)

Error return value of `c.AbortWithError` is not checked

(errcheck)